framework/test/unit/modules/interoperability/method.spec.ts

```diff
@@ -474,17 +462,25 @@ describe('Sample Method', () => {
 		it('should process with input chainID', async () => {
 			jest.spyOn(chainAccountStoreMock, 'has').mockResolvedValue(true);
 
-			await sampleInteroperabilityMethod.getMinReturnFeePerByte(methodContext, newChainID);
-			expect(channelStoreMock.get).toHaveBeenCalledWith(expect.anything(), newChainID);
+			await sampleInteroperabilityMethod['_getChannelCommon'](methodContext, newChainID);
+			expect(channelDataStoreMock.get).toHaveBeenCalledWith(expect.anything(), newChainID);
 		});
 
 		it('should throw error if channel is not found', async () => {
-			jest.spyOn(channelStoreMock, 'has').mockResolvedValue(false);
+			jest.spyOn(channelDataStoreMock, 'has').mockResolvedValue(false);
 
 			await expect(
-				sampleInteroperabilityMethod.getMinReturnFeePerByte(methodContext, newChainID),
+				sampleInteroperabilityMethod['_getChannelCommon'](methodContext, newChainID),
 			).rejects.toThrow('Channel does not exist.');
 		});
+
+		it('should return channel data for `getMessageFeeTokenID` & `getMinReturnFeePerByte`', async () => {
```

Comment:
maybe in this case you need to call these 2 functions to actually test if the `_getChannelCommon` is really called